markdown/reference/sql/CREATE-CAST.html.md.erb

```diff
@@ -0,0 +1,101 @@
+
+To be able to create a cast, you must own the source or the target data type. To create a binary-compatible cast, you must be superuser.
+
+
+
+## <a id="topic1__section4"></a>Parameters
+
+<dt>\<sourcetype\>  </dt>
+<dd>The name of the source data type of the cast.</dd>
+
+<dt>\<targettype\>  </dt>
+<dd>The name of the target data type of the cast.</dd>
+
+<dt> \<funcname(argtypes)\>   </dt>
+<dd>The function used to perform the cast. The function name may be schema-qualified. If it is not, the function is looked up in the schema search path. The function's result data type must match the target type of the cast.
+Cast implementation functions may have one to three arguments. The first argument type must be identical to the cast's source type. The second argument, if present, must be type `integer`; it receives the type modifier associated with the destination type, or `-1` if there is none. The third argument, if present, must be type `boolean`; it receives `true` if the cast is an explicit cast, `false` otherwise. In some cases, the SQL specification demands different behaviors for explicit and implicit casts. This argument is supplied for functions that must implement such casts. Designing  your own data types this way is not recommended.
+Ordinarily, a cast must have different source and target data types. However, declaring a cast with identical source and target types is allowed if it has a cast implementation function with more than one argument. This is used to represent type-specific length coercion functions in the system catalogs. The named function is used to coerce a value of the type to the type modifier value specified by its second argument. (Since the grammar presently permits only certain built-in data types to have type modifiers, this feature is of no use for user-defined target types.)
+When a cast has different source and target types and a function that takes more than one argument, it represents a conversion from one type to another and applying a length coercion in a single step. When no such entry is available, coercion to a type that uses a type modifier involves two steps, one to convert between data types and a second to apply the modifier.
+</dd>
+
+<dt>WITHOUT FUNCTION \<call\_handler\>   </dt>
+<dd>Indicates that the source type and the target type are binary compatible, so no function is required to perform the cast.</dd>
+
+<dt>AS ASSIGNMENT \<valfunction\>   </dt>
```

Comment:
Neither the syntax block nor text mentions \<valfunction\>.  This also doesn't appear in the GPDB version of this reference.